cloudpickle/cloudpickle.py

```diff
@@ -1295,20 +1340,17 @@ def _is_dynamic(module):
     return _find_spec(module.__name__, pkgpath, module) is None
 
 
-def _make_typevar(name, bound, constraints, covariant, contravariant,
-                  class_tracker_id):
-    tv = typing.TypeVar(
+def _make_typevar(name, bound, constraints, covariant, contravariant):
+    return typing.TypeVar(
         name, *constraints, bound=bound,
         covariant=covariant, contravariant=contravariant
     )
-    return _lookup_class_or_track(class_tracker_id, tv)
```

Comment:
Why don't you track `TypeVar` defintions anymore? This seems unrelated to the change to support early 3.5.x. 

Doesn't this change break ant test?

Edit: I see you removed `test_pickle_dynamic_typevar_tracking`.

Reply:
Also won't this break unpickling objects pickled with cloudpickle 1.4.0?

Reply:
`TyperVar` instances are not weakreferable in `Python 3.5.3`..

Reply:
> Also won't this break unpickling objects pickled with cloudpickle 1.4.0?

We can always restore the `class_tracker_id` for backward compat.

Reply:
> TyperVar instances are not weakreferable in Python 3.5.3..

I re-ran the tests with old Python 3.5.x and they pass...

Reply:
Hum you are right they fail just for 3.5.3 ...

Reply:
ok let me do a workaround for 3.5.3.